mail_tracking/__openerp__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Email tracking",
     "summary": "Email tracking system for all mails sent",
-    "version": "8.0.3.0.0",
+    "version": "8.0.3.0.1",
```

Comment:
@Yajo, we use this syntax: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers